File: engine/engine-model/src/main/scala/tools/packet/buff/GiveForeignChairSkillEffect.scala
package tools.packet.buff

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GiveForeignChairSkillEffect(private var _characterId: Int) extends PacketInput {
  def characterId: Int = _characterId

  override def opcode(): SendOpcode = SendOpcode.GIVE_FOREIGN_BUFF
}

File: engine/engine-model/src/main/java/tools/packet/buff/GiveForeignChairSkillEffect.java
package tools.packet.buff;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GiveForeignChairSkillEffect(Integer characterId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GIVE_FOREIGN_BUFF;
   }
}
